auslib/admin/views/forms.py

```diff
@@ -168,7 +169,7 @@ class PartialReleaseForm(Form):
 
 
 class RuleForm(Form):
-    backgroundRate = IntegerField('Background Rate', validators=[Required(), NumberRange(0, 100)])
+    backgroundRate = IntegerField('Background Rate', validators=[any_of(range(0, 101, 1))])
```

Comment:
No really, I forgot about that TBH cause of error "phantom" errors I was getting from test. I will commit now version with NumberRange and then we can alter error message to your liking.